packages/plugin-page-view-tracking-browser/src/typings/page-view-tracking.ts

```diff
@@ -1,5 +1,8 @@
 export interface PageTrackingBrowserOptions {
   trackOn?: PageTrackingTrackOn;
+  trackHistoryChanges?: PageTrackingHistoryChanges;
 }
 
 export type PageTrackingTrackOn = 'attribution' | (() => boolean);
+
+export type PageTrackingHistoryChanges = 'all' | 'pathOnly';
```

Comment:
im curious if we need to filter here. im thinking the popstate event and pushState method should be enough for us to tell if they navigate to a different page. i'd be curious if we can expand on PageTrackingTrackOn fn to work with this kind of filter

Reply:
So my thinking was SPA's often use query params or hashes to save things that update frequently like user searches. Implementers should probably be using replaceState not pushState for things like that most of the time though, but seems like an easy mistake to make. Overall I added this to make he implementer feel a little safer they're not going to blow up their event volume with meaningless views. You're right this wouldn't be hard to implement themselves in trackOn. It's definitely a nice to have.